added moovingon.ai tile (#2338)
DataDog/integrations-extras: moovingon_ai/README.md

# moovingon.ai

## Overview
[moovingon.ai](http://moovingon.ai/) revolutionizes the management of 24/7 operations for organizations through advanced AIOps and runbook automation. It serves start-ups, scale-ups, and global enterprises alike by seamlessly integrating into your infrastructure and simplifying the complexities of cloud operations management.

> With [moovingon.ai](http://moovingon.ai/), you can achieve unparalleled control and efficiency. It orchestrates monitoring, productivity, and escalation processes across your entire organization. Our technology adapts to your unique tech stack and workflows, providing a centralized hub for comprehensive observability and consistent operations. Experience over 90% automated healing to focus on strategic initiatives and reduce operational costs.

Reviewer: The overview shouldn't include any marketing copy. Instead, it should include: 
	1	What does your product do in 1-2 sentences?
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What specific data your integration monitors and the value?

## Setup

### Installation

> To install moovingon.ai Datadog integration, please log into the moovingon.ai platform:

Reviewer 2: ```suggestion
To install the moovingon.ai Datadog integration, log into the moovingon.ai platform, then follow the below steps:
```

1. Navigate to **Integrations** and select **Add Datadog**.

2. Enter a name for the integration and **Submit**.

3. Proceed to the Datadog OAuth2 screen and click on the **authorize** button.

> 4. **(Optional)** To receive events from all monitors, edit and click on **install/update the webhook**.

Reviewer: The authorization process should kick off from Datadog. Please update this to reflect the steps the customer will take within Datadog to when they click authorize

Author: I'm not entirely sure what you mean. In step 3 I specified that within datadog he has to click on "authorize". That's pretty much sums it up. There's nothing else he needs to do in Datadog.

Author: I assumed it's possible to move the user to moovingon.ai where he will add a datadog integration by using Oauth2. If it's not the case, where do I implement the logic to interact with my app? is it inside the tile?

# moovingon.ai

## Overview

> [moovingon.ai](http://moovingon.ai/) offers a platform to help organizations manage their 24/7 operations using automated processes and centralized monitoring management. The platform includes alert aggregation and correlation, automatically linked with runbooks and an analytics engine for reports and dashboards.

Reviewer: ```suggestion
[moovingon.ai][1] offers a platform to help organizations manage their 24/7 operations using automated processes and centralized monitoring management. The platform includes alert aggregation and correlation, automatically linked with runbooks and an analytics engine for reports and dashboards.
```

This integration will enable you to:

1. Consolidate all your datadog alerts under a centralized platform.
2. Automatically link incident remediation runbooks with alerts. 

> 3. Log all remediation activities in datadog events. 

Reviewer: The overview sounds like moovingon is only pulling data out of Datadog. Can you include what this integration is monitoring, what is being sent to Datadog, and the value?

Reviewer: Consolidate all your datadog alerts under a centralized platform.
- is this ocnsolidating moovingon alerts within Datadog? 

Automatically link incident remediation runbooks with alerts. 
- Where is this linked? is it moovingon incidents or Datadogs? Is it Datadog alerts or moovingon? 

Log all remediation activities in datadog events. 
- what does this mean? what are the remediation activities?

Author: This platform was made for NOC. Every manual or automated action that we run from moovingon.ai will send an audit to datadog events.

The result is that lets say if I run a webhook from my platfrom, and that webhook restart the host of the monitor I'm corrently look at in DataDog, your users will be able to see the audits of the actions we take from my end.

ex.
1. Datadog triggered alert.
2. Event is being created in moovingon.ai
3. NOC engineer is taking a manual action, lets say restart the host manual OR automated flow is running, one of the steps is to restart the host by running a webhook / jenkins action.
4. The audit of the action will be send back to the Datadog monitor as an event so the user could correlate the behavior of the test in Datadog based on the action we made in our end. (ex. the alert is showing red for 5 mins because of a restart action from moovingon.ai, without this info you won't know why that datadog monitor is failing)

Reviewer: Thanks for the insight! Can you add specifically what your product is doing like the above then what is being sent to Datadog so it's clear to users what's being sent to Datadog and what is being done within your product? The overview should be clear enough for someone who has no knowledge of our product and learn what your product does, what the integration does, what your integration specifically monitors, what is sent to Datadog.

Author: We've made some changes like you requested.

## Setup

### Installation

To install the moovingon.ai Datadog integration, log into the moovingon.ai platform, then follow the below steps:

1. Navigate to **Integrations** and select **Add Datadog**.

2. Enter a name for the integration and **Submit**.

3. Proceed to the Datadog OAuth2 screen and click on the **authorize** button.

4. **(Optional)** To receive events from all monitors, edit and click on **install/update the webhook**.

### Metrics

moovingon.ai does not include any metrics.

### Service Checks

moovingon.ai does not include any service checks.

> 

Reviewer: ```suggestion
### Events

The moovingon integration includes events.
```
Does your integration send in events? if so, please include the PR

# moovingon.ai

## Overview

> [moovingon.ai][1] is a platform for cloud operations and NOC management. It consolidates alerts across your observability suite and associates them with automated runbooks for alert and incident remediation. By integrating this, you can utilize the power of Datadog and moovingon.ai for efficient, automated incident management.

Reviewer: ```suggestion
[moovingon.ai][1] is a platform for cloud operations and NOC management. It consolidates alerts across your observability suite and associates them with automated runbooks for alert and incident remediation. With this integration, you can utilize the power of Datadog and moovingon.ai for efficient, automated incident management.
```

Author: Added

> moovingon.ai uses Datadog's monitors, logs, and event data for alert correlation and aggregation.

Reviewer: Can we change this to something like the below : 

This integration sends in moovingon.ai log and event data to Datadog such as " replace with types of data monitored". With this integration, you can utilize the out-of-the-box Datadog dashboard for alert correlation and aggregation.

Author: Not really, because it's pretty much what I wrote in line 10. 

"2. **Comprehensive incident management**: Log all remediation actions performed in moovingon.ai in Datadog events for compliance and remediation clarity."

These events are visible from the actual monitors that were triggered in Datadog or inside the moovingon.ai Overview dashboard.

Line 6 is referring to moovingon.ai itself so if I change it it's like I'm write the same thing twice. you can suggest a change for line 10 if you think it should change.

Reviewer: Thanks for clarifying! If the integration doesn't include an ootb monitor, we should be clear that they can create Datadog monitors to alert or remove monitors from the sentence

Author: @emarsha94 it's optional. The user can either mark all the monitors or select whatever monitors he wants. I wouldn't call it ootb because the user himself should decide if that's the behavior he wants or not. If you look at line 21, I already wrote about that:
4. **(Optional)** To receive events from all monitors, click on **install/update the webhook**.

Author: I rewritten step 4 to be more precise about what the user have to do.

Key features of this integration include:

1. **Focused Alert Management**: Bring all your Datadog alerts together in moovinon.ai easy-to-view dashboard, giving you simple control and clear visibility.

> 2. **Comprehensive incident management**: Log all remediation actions performed in moovingon.ai in Datadog events for compliance and remediation clarity.

Reviewer: Is this automatically logged? If so, we should be clear: 

"all remediation actions performed in moovingon.ai are sent to Datadog as events for compliance and remediation clarity "

> 3. **Extensive Analysis**: Utilize the analytics provided by moovingon.ai to derive insights from Datadog alerts. This assists in proactive decision-making and trend analysis.

Reviewer: Could you clarify what this means? Are you pulling alerts from Datadog and sending them to moovingon.ai for customers to see or is this within Datadog?

Author: 1. Yes, within moovingon.ai, we already talked about it [here](https://github.com/DataDog/integrations-extras/pull/2338#discussion_r1548152859) 
2. I'm pushing data to datadog so it's witin Datadog
3. Within Datadog

Reviewer: 1. This should be clear if this is within moovingon.ai or Datadog. If you are pulling all Datadog alerts and sending to moovingon.ai and saying they can view all of this within moovingon.ai then let's clarify that. To clarify, since there is an out of the box dashboard for the integration in datadog and a dashboard in moovingon.ai  this might be confusing on which one you are referring to. Ex:

> Bring all your Datadog alerts together in moovinon.ai easy-to-view dashboard, giving you simple control and clear visibility.

If you are referring to a dashboard within moovingon.ai: 
--->  Within moovingon.ai, use the moovingon.ai dashboard to bring all your Datadog alerts together giving you simple control and clear visibility.

If you are referring to the out of the box moovingon.ai dashboard within Datadog: 
---> Use the out-of-the-box Dashboard within Datadog to bring all of your alerts together giving you control and clear visibility 

 
3. Can you clarify for 3 is within Datadog? You can just add " from Datadog alerts within Datadog"

Reviewer: Can you implement the above? Let me know if you have questions on the changes requested

Author: Fair points, I added the changes above.

## Setup

### Installation

1. Click on **Connect Accounts** in order to login into moovingon.ai.
2. Enter a name for the datadog integration and **Submit**.

> 3. Proceed to the Datadog OAuth2 screen and click on the **authorize** button.

Reviewer 2: Is this in lowercase in the UI?

Author: It's not, fixed.

> 3. Proceed to the Datadog OAuth2 screen and click on the **authorize** button.

Reviewer 2: Same question as above: Is this in lowercase in the UI?

Author: Nope, fixed

> 4. **(Optional)** To receive events from all monitors, click on **install/update the webhook**.

Reviewer 2: Also this

Author: Fixed

## Uninstallation

1. Inside the moovingon.ai account, Go to **Settings** --> **Templates** and remove all the related datadog templates.
2. Go to **Setings** --> **Integrations** and remove the datadog integration.
3. Inside Datadog, **Integrations**  --> **Integrations**.
4. Click the moovingon.ai tile and click on **Uninstall integration**.

### Metrics

moovingon.ai does not include any metrics.

### Service Checks

moovingon.ai does not include any service checks.

> ### Events

Reviewer: ```suggestion

### Events
```

Reviewer: Sorry - looks like we accidentally got rid of both spaces instead of just the extra space. Adding one space between them!

# moovingon.ai

## Overview
[moovingon.ai][1] is a platform for cloud operations and NOC management. It consolidates alerts across your observability suite and associates them with automated runbooks for alert and incident remediation. By integrating this, you can utilize the power of Datadog and moovingon.ai for efficient, automated incident management.

moovingon.ai uses Datadog's monitors, logs, and event data for alert correlation and aggregation.
Key features of this integration include:

1. **Focused Alert Management**: Bring all your Datadog alerts together in moovinon.ai easy-to-view dashboard, giving you simple control and clear visibility.
2. **Comprehensive incident management**: Log all remediation actions performed in moovingon.ai in Datadog events for compliance and remediation clarity.
3. **Extensive Analysis**: Utilize the analytics provided by moovingon.ai to derive insights from Datadog alerts. This assists in proactive decision-making and trend analysis.

> 
> 

Reviewer: ```suggestion
```
